utils/main.go

```diff
@@ -88,6 +89,20 @@ func CheckExpired(expireOn string) (bool, error) {
 
 }
 
+// ValidateUserKey checks for the presence of a characters which should not be
+// allowed in a username string, returns an error if any such characters are
+// detected, nil otherwise.
+// Currently, the only banned characters are whitespace characters.
+func ValidateUserKey(user string) (error) {
+	whiteSpaceRegexp, _ := regexp.Compile("\\s")
```

Comment:
Do you need to use `regexp` for this purpose? Wont `strings.Contains()` be more efficient?

Reply:
Does strings.Contains() have a way to check for any whitespace character? 

Strictly speaking, it seems like anything which falls under unicode.IsSpace() should not be in the user string.

Reply:
I see, looks good then.